src/pages/QuestionPage.tsx

```diff
@@ -78,45 +154,54 @@ export default function QuestionPage() {
       justifyContent="center"
     >
       <Flex alignItems="center" maxW="md" margin="4">
-        {gotQuestion && prompt && answers && index ?
+        {prompt && answers && index ?
           <VStack>
             <Logo size={["32px", "50px"]} />
+            {isHost && <GameCode id={gameCode}></GameCode>}
+            <Timer index={index}/>
             <Question
               id="1"
               title={`Question ${index ? `#${index}` : ""}`}
               text={prompt ? prompt : ""}
             />
             <Answer setAnswer={handleSetAnswer}
               id="0"
-              background="red"
-              opacity={answer === 0 ? "100%" : "50%"}
+              background={answerArr[0]}
+              selected={selectedAnswer(answer, 0)}
+              disabled={answered}
               text={answers?.at(0)} />
             <Answer setAnswer={handleSetAnswer}
               id="1"
-              background="blue"
-              opacity={answer === 1 ? "100%" : "50%"}
+              background={answerArr[1]}
+              selected={selectedAnswer(answer, 1)}
+              disabled={answered}
               text={answers?.at(1)} />
             <Answer setAnswer={handleSetAnswer}
               id="2"
-              background="green"
-              opacity={answer === 2 ? "100%" : "50%"}
+              background={answerArr[2]}
+              selected={selectedAnswer(answer, 2)}
+              disabled={answered}
               text={answers?.at(2)} />
             <Answer setAnswer={handleSetAnswer}
               id="3"
-              background="orange"
-              opacity={answer === 3 ? "100%" : "50%"}
+              background={answerArr[3]}
+              selected={selectedAnswer(answer, 3)}
+              disabled={answered}
               text={answers?.at(3)} />
             {isHost ?
-              <Button onClick={nextQuestion} alignSelf="end" fontWeight="extrabold" shadow="lg" border="4px">Next Question</Button>
+              timeExpired &&
```

Comment:
I think for the host, the next question and show answer buttons should not be disabled with the timer. When the host rejoins via cookies, the timer resets back to 60 seconds so they would have to wait 60 seconds to move on to the next question.

In general, I think the host actions should not be limited as they are expected to be the game masters.